File: Project/C++/Project1/Pen.cpp
#include "Pen.h"
int Pen::ammount = 0;

Pen::Pen(std::string name, double price, std::string description, int stock, int id, std::string color, double tipSize)
{
	setName(name);
	setPrice(price);
	setDescription(description);
	setStock(stock);
	setId(id);
	setObjType("pen");
	this->color = color;
	this->tipSize = tipSize;
	this->ammount++;
}

std::string Pen::getDetails()
{
	return "Color: " + color + ". Tip Size: " + std::to_string(tipSize) + ".";
}

File: Project/Java/src/eshop/Pen.java
package eshop;

public class Pen extends Item{
    private String color;
    private double tipSize;
    private static int count = 0;
    Pen(String name, double price, String description, int stock, int id,String color, double tipSize){
        setName(name);
        setPrice(price);
        setDescription(description);
        setStock(stock);
        setId(id);
        this.color = color;
        this.tipSize = tipSize;
        count++;
    }

    public String getDetails(){
        return "Color: " + color + ", Tip size: " + tipSize;
    }
    public static int getCount() {
        return count;
    }
}
